src/components/UiFooter/DefaultFooter/DefaultFooter.tsx

```diff
@@ -27,7 +27,7 @@ function DefaultFooter({
   );
 
   return (
-    <Stack sx={styles.footerWrapper}>
+    <Stack sx={styles.footerWrapper} data-testid="default-footer">
```

Comment:
on the last call, i explained that we need to remove all data-testid from everywhere

Reply:
I did that because we also decided that it is not a priority at the moment, and I don't know the solution yet, how to take some elements without testId